ownCloud/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -0,0 +1,306 @@
+//
+//  PasscodeViewController.swift
+//  ownCloud
+//
+//  Created by Javier Gonzalez on 03/05/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+
+typealias PasscodeHandler = (() -> Void)
+
+let numberDigitsPasscode = 4
+let passcodeKeychainAccount = "passcode-keychain-account"
+let passcodeKeychainPath = "passcode-keychain-path"
```

Comment:
I made private the `numberDigitsPasscode` because the `passcodeKeychainAccount` and `passcodeKeychainPath` are used also on the `UnlockPasscodeManager`.

We should have a constants file to store the keys for Keychain of the app and also for the UserDefauls. What do you think? @felix-schwarz